components/infobox/wikis/ageofempires/infobox_person_player_custom.lua

```diff
@@ -74,6 +76,23 @@ local INACTIVITY_THRESHOLD_BROADCAST = {month = 6}
 function CustomPlayer.run(frame)
 	_player = Player(frame)
 	_args = _player.args
+	
+	-- Automatic team, history
+	if String.isEmpty(_args.team) then
+		_args.team = PlayerTeamAuto._main{team = 'team'}
+	end
+	automatedHistory = TeamHistoryAuto.results{player=_player.pagename, convertrole=true, addlpdbdata=true}
+	if String.isEmpty(_args.history) then
+		_args.history = automatedHistory
+	else
+		_args.history = tostring(mw.html.create('div')
+			:addClass("show-when-logged-in")
```

Comment:
```suggestion
			:addClass("user-is-editor")
```

Reply:
This does not seem to work: https://liquipedia.net/ageofempires/DauT

Reply:
would need a new class
atm afaik there is only
`show-when-logged-in` and `show-when-admin`

Reply:
Ah, i'm fine with the logged-in users then.